compiler/test/runner.re

```diff
@@ -271,12 +277,16 @@ let makeParseRunner =
           comments: List.map(comment_loc_stripper, comments),
           prog_loc: Location.dummy_loc,
         };
-      let parsed = strip_locs @@ parseString(name, input);
-      let untagged = strip_locs @@ parsed;
+      let parsed =
+        if (keep_locs) {
+          parseString(name, input);
+        } else {
+          strip_locs @@ parseString(name, input);
```

Comment:
Just curious, why do we strip the locs at all here?

Reply:
For most of the parsing tests, we just care about the nodes in the AST and it'd be a huge PITA to _also_ have to figure out the locations of everything, especially since the locations are all character-based instead of line/column based. You can't even write a helper to convert a line/column pair into this kind of loc since it depends on what's come before it in the file 😩